packaging/installer/README.md

```diff
@@ -293,6 +293,28 @@ To apply the changes you made, you have to restart Netdata.
 
 ---
 
+### Binary Packages
+![](https://raw.githubusercontent.com/netdata/netdata/master/web/gui/images/packaging-beta-tag.svg?sanitize=true)
+
+We provide our own flavour of binary packages for the most common operating systems that comply with .RPM and .DEB packaging formats.
+We have currently released .RPM versions with version [1.16.0](https://github.com/netdata/netdata/releases/tag/v1.16.0). We are planning to release packages following the .DEB format on one of the following releases. Our current packaging infrastructure provider is [Package Cloud](https://packagecloud.io).
+Netdata is committed to support installation of our solution to all operating systems.
+This is a constant battle for Netdata, as we strive to automate and make things easier for our users.
+For the operating system support matrix, please visit our [distributions](../DISTRIBUTIONS.md) support page.
+
+We provide two separate repositories, one for our stable releases and one for our nightly releases.
+1. Stable releases
```

Comment:
You need another return before the ordered list. On the preview it's combined with the sentence just above.